anchore_engine/services/policy_engine/api/controllers/feeds.py

```diff
@@ -220,14 +239,20 @@ def delete_feed(feed):
 
 @authorizer.requires_account(with_types=INTERNAL_SERVICE_ALLOWED)
 def delete_group(feed, group):
-    if feed == GRYPE_DB_FEED_NAME:
+    provider = get_vulnerabilities_provider()
+    internal_feed_name = provider.display_mapper.get_internal_name(feed)
+    if isinstance(provider, GrypeProvider) and internal_feed_name == GRYPE_DB_FEED_NAME:
         raise HTTPNotImplementedError(
-            message="Deleting individual groups for the grypedb feed is not yet supported.",
+            message="Deleting individual groups for the {} feed with the grype vulnerability provider enabled is not yet supported.".format(
```

Comment:
Same comment as above re string formatting.

Reply:
switched to f-strings